### PR TITLE
Np globe v2

### PR DESCRIPTION
**Pull request by lcaloguerea**
[NP-6](https://pappcorn.atlassian.net/browse/NP-6)

# Change description
- Added bottom bar with toaster message for pending features

# Evidence
imagen

Static Image, Gif or Videos that show the change in the pr 
imagen

### CODE REVIEW

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 1]
```diff
@@ -76,35 +92,59 @@ export class GlobeComponent implements OnInit {
...
-    d3.select(self.frameElement).style('height', this.height + 'px');
-    //lets do it globe end!
+  ngOnInit() {
+    this.isLoading = true;
+
+    setTimeout(() => {
+      this.isLoading = false;
+      this.hidden = false;
+    }, 4000);
+    this.usersSubscription = this.userService.getAll().subscribe((users) => {
```
Comment (ruedagato) [inline apps/mobile/src/app/home/components/globe/globe.component.ts:109]: we have to search for other solution to don't get all user or maybe get the user for algolia and not for firestore

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 2]
```diff
@@ -240,15 +284,21 @@ export class GlobeComponent implements OnInit {
...
+      d3.json('../../../../assets/globe-data/locations.json').then((data2) => {
+        this.locations = data2;
+      });
 
       window.addEventListener('resize', this.scale);
       this.scale();
     });
   }
+
+  ngOnDestroy() {
```
Comment (ruedagato) [inline apps/mobile/src/app/home/components/globe/globe.component.ts:299]: add implement onDestory on class

File: apps/mobile/src/assets/globe-data/locations.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (ruedagato) [inline apps/mobile/src/assets/globe-data/locations.json:1]: is this mock data?

File: functions/src/user-update.function.ts [hunk 4]
```diff
@@ -41,9 +41,9 @@ const difference = (object: any, base: any) => {
     return _.transform(object2, (result: any, value: any, key: string) => {
       if (!_.isEqual(value, base2[key])) {
         result[key] =
-          _.isObject(value) && _.isObject(base2[key]) ?
-            changes(value, base2[key]) :
-            value;
+          _.isObject(value) && _.isObject(base2[key])
+            ? changes(value, base2[key])
```
Comment (ruedagato) [inline functions/src/user-update.function.ts:45]: don't change it, the eslint for function generate an error

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 5]
```diff
@@ -9,10 +12,14 @@ import * as versor from 'versor';
   styleUrls: ['./globe.component.scss'],
 })
 export class GlobeComponent implements OnInit {
```
Comment (ruedagato) [inline apps/mobile/src/app/home/components/globe/globe.component.ts:11]:
```suggestion
 export class GlobeComponent implements OnInit, OnDestroy {
```

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 6]
```diff
@@ -76,35 +92,59 @@ export class GlobeComponent implements OnInit {
...
-
-    d3.select(self.frameElement).style('height', this.height + 'px');
-    //lets do it globe end!
+  ngOnInit() {
+    this.isLoading = true;
+
+    setTimeout(() => {
+      this.isLoading = false;
+      this.hidden = false;
+    }, 4000);
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/mobile/src/app/home/components/globe/globe.component.ts]: Solved

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 7]
```diff
@@ -45,6 +59,8 @@ export class GlobeComponent implements OnInit {
   a0;
   c;
   l;
+  coords: any;
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/mobile/src/app/home/components/globe/globe.component.ts:62]: Can't do it, D3 conflict with this

File: apps/mobile/src/app/home/components/globe/globe.component.ts [hunk 8]
```diff
@@ -9,10 +12,14 @@ import * as versor from 'versor';
   styleUrls: ['./globe.component.scss'],
 })
 export class GlobeComponent implements OnInit {
+  usersSubscription: Subscription;
+  users: User[] = [];
+  schoolLocations: any[];
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/mobile/src/app/home/components/globe/globe.component.ts:17]: Can't do it, D3 conflict with this

**Review by ruedagato — APPROVED** [review]

**Pull request merged**